tests/test_errors.py

```diff
@@ -31,6 +30,21 @@ def test_error_traceback(program):
         http('--traceback', 'www.google.com')
 
 
+@mock.patch('httpie.core.program')
+def test_connection_error(program):
+    exc = ConnectionError('Connection failed')
+    exc.request = Request(method='GET', url='http:/www.httpies.org')
```

Comment:
Try using an address that clearly can never possibly exist, like something under the '.invalid' top-level domain from RFC 2606